src/otx/recipe/_base_/data/mmdet_base.yaml

```diff
@@ -24,6 +24,8 @@ config:
       - prob: 0.5
         type: RandomFlip
       - type: PackDetInputs
+    sampler:
+      class_path: torch.utils.data.RandomSampler
```

Comment:
I thought that base sampler of OTX1.x is BalancedSampler, if so could you explain the reason why we change default sampler type in OTX2.x?

Reply:
I'm going to merge this PR first, then apply `BalancedSampler` to Regression Test to check the performance, and if all goes well after that, I'll apply it by default.